test/qctrl/test_qctrl.py

```diff
@@ -28,6 +39,91 @@
 DIFFERENCE_THRESHOLD = 0.35
 
 
+class TestV2PrimitivesQCTRL(IBMIntegrationTestCase):
+    """Integration tests for V2 primitives using QCTRL."""
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.bell = bell()
+        self.backend = "alt_canberra"
+
+    @run_integration_test
+    def test_sampler_v2_qctrl(self, service):
+        """Test qctrl bell state with samplerV2"""
+        shots = 1000
```

Comment:
Let's save some QPU time if we are not verifying result data

```suggestion
        shots = 1
```